.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: true
+dist: trusty
+
+language: php
+
+services:
+  - mongodb
+
+php:
+  - hhvm-3.12
+  - hhvm-3.15
+
+matrix:
+  allow_failures:
+    - php: hhvm-3.12
+    - php: hhvm-3.15
```

Comment:
https://docs.travis-ci.com/user/notifications#Configuring-email-notifications seems like a better solution for that.